output/cloud/output.go

```diff
@@ -363,24 +364,33 @@ func (out *Output) AddMetricSamples(sampleContainers []metrics.SampleContainer)
 				values[metrics.IterationsName] = 1
 			}
 
+			encodedTags, err := easyjson.Marshal(sc.GetTags())
+			if err != nil {
+				out.logger.Error("Encoding tags failed", err)
```

Comment:
A small one otherwise, I believe formatting will be not so nice

```suggestion
				out.logger.WithError(err).Error("Encoding tags failed")
```

Reply:
I see no changes :cry:

Reply:
@olegbespalov I missed one, you're right, they are two lines in two different places. It should be fixed now.